src/libraries/Directory.Build.targets

```
@@ -127,6 +127,7 @@
   <Import Project="$(RepositoryEngineeringDir)testing\coverage.targets" Condition="'$(EnableRunSettingsSupport)' == 'true' or '$(EnableCoverageSupport)' == 'true'" />
 
   <Import Project="$(RepositoryEngineeringDir)illink.targets" Condition="'$(IsSourceProject)' == 'true' or '$(ExplicitlyImportCustomILLinkTargets)' == 'true'" />
+  <Import Project="$(RepositoryEngineeringDir)liveILLink.targets" />
```

Would it make sense to only import this file when the illink.targets file is imported?

```suggestion
  <ImportGroup Condition="'$(IsSourceProject)' == 'true' or '$(ExplicitlyImportCustomILLinkTargets)' == 'true'">
    <Import Project="$(RepositoryEngineeringDir)illink.targets" />
    <Import Project="$(RepositoryEngineeringDir)liveILLink.targets" />
  </ImportGroup>
```